Library/Formula/pidgin.rb

```diff
@@ -49,28 +52,29 @@ def install
       --disable-gstreamer
       --disable-gstreamer-interfaces
       --disable-gtkspell
-      --disable-idn
       --disable-meanwhile
       --disable-vv
     ]
 
-    args << "--with-x" if build.with? "GUI"
     args << "--disable-perl" if build.without? "perl"
     args << "--enable-cyrus-sasl" if build.with? "gsasl"
 
-    if build.without? "GUI"
+    if build.without? "gui"
       args << "--with-tclconfig=#{MacOS.sdk_path}/usr/lib"
       args << "--with-tkconfig=#{MacOS.sdk_path}/usr/lib"
       args << "--without-x"
       args << "--disable-gtkui"
+    else
+      args << "--with-x"
+      args << "--disable-idn"
     end
 
     system "./configure", *args
     system "make", "install"
 
-    if build.with? "GUI"
+    if build.with? "gui"
       resource("pidgin-otr").stage do
-        ENV.prepend "CFLAGS", "-I#{HOMEBREW_PREFIX}/opt/libotr/include/libotr"
+        ENV.prepend "CFLAGS", "-I#{Formula["libotr"].opt_include}"
```

Comment:
Seems to be finding it fine without the extra poke to the actual directory inside ` include `. Most things do. Not sure why we had it like this in the first place... Probably was my screw up :smile_cat: